### DISCUSSION TITLE
Quasar Firebase Extension is broken

### DISCUSSION THREAD

**Discussion by dragonballa** [quasarframework/quasar#13919] [General - Components / Directives / etc]
Followed the youtube video and also the suggested quasar ext add @vueauth/auth
 comment at https://www.youtube.com/watch?v=3P7ERhDLALM but still get error:

 App • Running "firebase" Quasar App Extension...

 App • ⚠️  Extension(firebase): Dependency not found - @quasar/app. Please install it.

I tried it via yarn and npm both fail. I tried adding firebase and firebase-composables. 

really looking forward to using Quasar but seems like some information/instructions are not working.

**Comment by metalsadman** [comment]
that extension is still a WIP based on the repo https://github.com/ldiebold/quasar-ui-firebase, you can file an issue there for Luke to see.
